files/usr/lib/jeos-firstboot

```diff
@@ -122,8 +122,11 @@ d(){
 	retval=
 	while true
 	do
+        # disable exit on error for capturing dialog return codes
+	        set +e
```

Comment:
```
retval=0
dialog --backtitle "$PRETTY_NAME" --output-fd 3 "$@" 3>"${dialog_out}" || retval=$?
```

Reply:
Stylistic choice right?

Reply:
Could be argued, but changing the shell mode should be avoided.